internal/local/command_kafka_start.go

```diff
@@ -256,16 +216,16 @@ func (c *Command) prepareAndSaveLocalPorts(cmd *cobra.Command, isTest bool) erro
 			PlaintextPorts:  []string{"3002", "3003"},
 		}
 	} else {
-		freePorts, err := freeport.GetFreePorts(6)
+		freePorts, err := freeport.GetFreePorts(int(3 * numOfBrokers))
```

Comment:
Why `* 3`? Does every broker have its own Kafka REST instance? That shouldn't be the case... right?

Reply:
you're right. only one krest port is needed.

Reply:
will update

Reply:
krest is always 8082, but controller / broker / plaintext are 3 different ports